frontend/src/components/layout/navigation/whatsnew/WhatsNewMenu.test.tsx

```diff
@@ -25,6 +25,14 @@ const l10nMock = {
   ),
 };
 
+const runtimeDataWithNoBundle = {
```

Comment:
suggestion (non-blocking): 
```suggestion
const mockedRuntimeDataWithNoBundle = {
```